core/src/validator.rs

```diff
@@ -1107,7 +1110,7 @@ impl Validator {
                 connection_cache.clone(),
                 max_complete_transaction_status_slot,
                 max_complete_rewards_slot,
-                prioritization_fee_cache.clone(),
+                prioritization_fee_cache.clone().unwrap(),
```

Comment:
I think ideally, we would not have to unwrap here.
`prioritization_fee_cache` can/should be initialized in the same if statement...however it involves re-ordering a lot of stuff so that RPC is spawned earlier on.

Thinking it's simpler for this PR to just unwrap, and a follow-up PR for re-ordering initializaton in `validator.rs`?